CMakeLists.txt

```diff
@@ -445,7 +445,6 @@ if(EMSCRIPTEN)
     # TODO: Remove the exported runtime methods
     # after the next xeus release.
     target_link_options(xcpp
-        PUBLIC "SHELL: -s EXPORTED_RUNTIME_METHODS='[\"FS\",\"PATH\",\"LDSO\",\"loadDynamicLibrary\",\"ERRNO_CODES\"]'"
```

Comment:
This is now being enforced through 

https://github.com/jupyter-xeus/xeus/blob/f1808d91b117dc337b6edf2ad5f709e41836c4b6/cmake/WasmBuildOptions.cmake#L38